jinahub/rankers/DPRReaderRanker/dpr_reader.py

```diff
@@ -0,0 +1,231 @@
+from itertools import groupby
+from typing import Dict, Iterable, List, Optional, Tuple, Union
+
+import numpy as np
+import torch
+from jina import Document, DocumentArray, Executor, requests
+from transformers import DPRReader, DPRReaderTokenizerFast
+
+
+def _batcher(iterable, n=1):
```

Comment:
batcher functionality is already standarized from jina commons, u can use that one